.gitmessage.txt

```diff
@@ -0,0 +1,11 @@
+Subject line (try to keep under 50 characters)
+
+{current situation} -- use present tense}
+
+{why it needs to change}
+
+{what is being done about it} -- use imperative mood}
+
+{why it is done that way}
+
+{any other relevant info}
```

Comment:
now that this file is `.gitignore`d, you can `git rm` this file to remove this file from the repository (but will still be on your local machine)